### ISSUE TITLE
Don't allow users to attempt NO DATA filling with LCP files in the GUI

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#261]
A user just reported a case where he loaded an lcp from WFDSS and the GUI warned him about NO DATA values in the file and offered to auto fill. When he chose yes, the file browser opened to let him rename the file (but it will only generate a tif). Since we don't auto-fill lcp files, we shouldn't offer this auto-fill option in the GUI. Or we should auto-fill the lcp files correctly. One or the other.

**Comment by ksshannon** [comment]
We can't fill classified bands, right?  Seems like we just reject it.

**Comment by jforthofer** [comment]
Good point.  We'd either need to write an interpolation/extrapolation method for classified bands or just reject.

**Comment by ksshannon** [comment]
I don't see how we do that with something like fuel models, without making huge assumptions.

**Comment by ksshannon** [comment]
We could interpolate the DEM, and fall back on grass/brush/trees selection from the user for roughness.

**Comment by nwagenbrenner** [comment]
The user in this case had an edited LCP (some of the vegetation bands were modified to reflect more current conditions). It seems wrong to offer an interpolation method that completely discounts this effort. The fact that perhaps it makes little/no difference (depending on which solver is used) in the actual solution is something we should address at some point in the code, but is probably not obvious to the user. I think if they supply an LCP, we should use it, or reject it. Or, at the least, we report the issue (summarized in this ticket, including the bit about the vegetation not impacting the solution significantly) and tell them we will interpolate the elevation, but nothing else.

**Comment by jforthofer** [comment]
We could make an interpolation/extrapolation method for classified layers that does something like checking the 8 neighbor cells and picking the value that is most numerous ("most likely" classified value).  So for example if, of the 8 neighbor cells, 5 are fuel model 10, and 3 are fuel model 8, we would make the cell fuel model 10, which is the most numerous value.  Chuck also agrees with this approach.

**Comment by ksshannon** [comment]
What's the tie breaker?  Randomly choose one?

**Comment by ksshannon** [comment]
We need a reliable way to write LCP files (GDAL write support would be easiest).  Moving to wishlist.

**Comment by jforthofer** [comment]
Yeah, I think in the case of a tie we just randomly choose one.

**Comment by nwagenbrenner** [comment]
This looks to be done in master. @latwood let me know if you think there is still more work to be done on this.
